openshift/configmap-template.yaml

```diff
@@ -125,7 +126,12 @@ parameters:
     value: "pushgateway"
 
   - displayName: PROMETHEUS_PUSHGATEWAY_PORT
-    description: A port to send prometheus metrics via pushgateway
+    description: A port to send prometheus metrics via pushgateway.
     name: PROMETHEUS_PUSHGATEWAY_PORT
     required: false
     value: "80"
+
+  - displayName: SENTRY_DSN
```

Comment:
what is a DSN? any link to doc?does the thoth-station org get a DSN? or a bot user?

Reply:
Updated.